cartography/intel/bitbucket/members.py

```diff
@@ -0,0 +1,68 @@
+import requests
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+from requests.exceptions import RequestException
+import neo4j
+from cartography.util import make_requests_url
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+logger = logging.getLogger(__name__)
+
+@timeit
+def get_workspace_members(access_token:str,workspace:str):
+    url = f"https://api.bitbucket.org/2.0/workspaces/{workspace}/members"
+    return make_requests_url(url,access_token)
+
+
+
+    
+def load_memebers_data(neo4j_session: neo4j.Session,members_data:List[Dict],common_job_parameters:Dict):
```

Comment:
use transaction base approach to load resources 
https://github.com/Cloudanix/cartography/blob/54a7e1ea0f3cd0522fee67e32ad08bab883ae8c8/cartography/intel/azure/aks.py#L21

Reply:
used transaction based approach